x-pack/plugins/monitoring/server/routes/api/v1/_health/index.ts

```diff
@@ -53,11 +50,19 @@ export function registerV1HealthRoute(server: MonitoringCore) {
         logger,
       };
 
+      const monitoringIndex = [
+        getNewIndexPatterns({ config, moduleType: 'elasticsearch' }),
+        getNewIndexPatterns({ config, moduleType: 'kibana' }),
+        getNewIndexPatterns({ config, moduleType: 'logstash' }),
+        getLegacyIndexPattern({ config, moduleType: 'beats' }),
```

Comment:
I think the plan is to rename `getNewIndexPatterns` to `getIndexPattern`. Should we use `getNewIndexPatterns` and just adapt it to decide when the new index pattern needs to be build (kibana, es, logstash)?

Reply:
Sounds good to me, do we a ticket that tracks this ?

Reply:
I think it's tracked here https://github.com/elastic/kibana/issues/123508